app/features/executive/business-continuity.feature

```diff
@@ -0,0 +1,13 @@
+@mvp
```

Comment:
Stretch goal for MVP -- let's try to get this done but recognize that we may have to punt some scenarios.